react/src/components/KeypairResourcePolicyInfoModal.tsx

```diff
@@ -0,0 +1,194 @@
+import { filterEmptyItem } from '../helper';
+import { useSuspendedBackendaiClient } from '../hooks';
+import AllowedVfolderHostsWithPermissionModal from './AllowedVfolderHostsWithPermissionModal';
+import BAIModal, { BAIModalProps } from './BAIModal';
+import Flex from './Flex';
+import ResourceNumber from './ResourceNumber';
+import { KeypairResourcePolicyInfoModalFragment$key } from './__generated__/KeypairResourcePolicyInfoModalFragment.graphql';
+import { Descriptions, Typography } from 'antd';
+import { createStyles } from 'antd-style';
+import { DescriptionsItemType } from 'antd/es/descriptions';
+import graphql from 'babel-plugin-relay/macro';
+import dayjs from 'dayjs';
+import _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { useFragment } from 'react-relay';
+
+const useStyles = createStyles(({ css }) => ({
+  description: css`
+    .ant-descriptions-row {
+      border: none !important;
+    }
+    .ant-descriptions-view > table {
+      width: fit-content !important;
+    }
+  `,
+}));
+
+interface InfoModalProps extends BAIModalProps {
+  open: boolean;
+  onRequestClose: () => void;
+  resourcePolicyFrgmt: KeypairResourcePolicyInfoModalFragment$key | null;
+}
+
+const KeypairResourcePoliciesInfoModal: React.FC<InfoModalProps> = ({
```

Comment:
File name is `KeypairResourcePolicyInfoModal` but component name is `KeypairResourcePoliciesInfoModal`. `Plolicy` and `Plicies`